lib/ontologies_api_client/request_federation.rb

```diff
@@ -1,9 +1,12 @@
 require 'active_support/core_ext/hash'
+require 'active_support/cache'
 
 module LinkedData
   module Client
     module RequestFederation
 
+      CACHE = ActiveSupport::Cache::MemoryStore.new
```

Comment:
this will not work as expected as in the Rails we don't use MemoryStore, but MemeCache or Redis.